codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyHeaders.java

```diff
@@ -150,11 +150,18 @@ private HttpNames() { }
     SpdyHeaders setObject(CharSequence name, Object... values);
 
     @Override
-    SpdyHeaders set(TextHeaders headers);
+    SpdyHeaders set(Headers<? extends CharSequence> headers);
 
     @Override
-    SpdyHeaders setAll(TextHeaders headers);
+    SpdyHeaders setAll(Headers<? extends CharSequence> headers);
 
     @Override
     SpdyHeaders clear();
+
+    /**
+     * {@link #get(CharSequence)} and convert the result to a {@link String}.
+     * @param name the name of the header to retrieve
+     * @return the first header value if the header is found. {@code null} if there's no such header.
+     */
+    String getAsString(CharSequence name);
```

Comment:
The problem with the static utility methods are that it makes the resulting code look ugly. e.g. `headers.getAllAsString()` vs. `SomeUtil.getAllAsString(headers)` vs. `headers.getAll()`.  We could solve this issue eventually when we require Java 8, but we are not there yet..

Previously, we had `getInt()`, `getDate()` and many other static utility methods for accessing text headers, but we eventually moved them into `Headers`.